unreleased/TEMPLATE.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type:
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component:
+
+# A single sentence describing the change
+note:
```

Comment:
Sometimes links are added to notes.  It would be nice to validate the links work before the changelog is generated as part of the release process.  Would a CI step need to generate the changelog on every PR so that check-links works or can we check the links without generating anything?